sec_certs_page/__init__.py

```diff
@@ -199,6 +201,21 @@ def get_searcher():
     app.register_blueprint(docs)
     app.register_blueprint(about)
     app.register_blueprint(chat)
+    
+    # Setup GitHub OAuth if available and configured
+    try:
```

Comment:
Assume Flask-Dance is always installed and instead add a config option that controls whether Github auth is enabled or not.